### PR TITLE
added eval ipynb

### PR DESCRIPTION
**Pull request by thejerrytan**
Added a jupyter notebook to visualize data, and do sanity check for trained models.

This is not ready to be merged yet.

I am including it first because it may be helpful to some of you who wants to evaluate your trained DL models.

If you want to run it with @Elgeweily 's ssd_classifier model, merge his PR branch in here.

### HOW DO I SET UP VALIDATION DATASET ###

Now to get the validation set on your own:

1) download from this link: https://github.com/coldKnight/TrafficLight_Detection-TensorFlowAPI#get-the-dataset
2) copy paste to ./data folder
3) the downloaded file name is dataset-sdcnd-capstone.zip. unzip. Root folder is dataset-sdcnd-capstone
4) assuming you are at projectRoot/data: `mv dataset-sdcnd-capstone/data ../validation`

### CODE REVIEW

File: dev-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+jupyter
```
Comment (ser94mor) [inline dev-requirements.txt]: I suggest to created conda env instead. It is more convenient and there you can use a Python 3. Just put the same into the `enviromnet.yml` file.

Comment (thejerrytan) [reply dev-requirements.txt]: done

**Comment by ser94mor** [comment]
This pull request **introduces 2 alerts** when merging 44de9c165da7285ec86420726543f3a61c88f4d4 into d4111bf44c893697599bf4ccdc5c2c08d9389c09 - [view on LGTM.com](https://lgtm.com/projects/g/aoool/self-driving-car-using-ros/rev/pr-c41be9cb230c9f552c6f87b005d4dc8d4fedd4bc)

**new alerts:**

* 2 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: ros/src/tl_detector/helpers.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import tensorflow as tf
```
Comment (ser94mor) [inline ros/src/tl_detector/helpers.py:1]: This file is no longer exist in `master`. I suggest just move `eval.ipynb` and `environment.yaml` to a new branch, started from the current `master` branch. It seems like your branch is merged with the old SSD Classifier branch that is not currently in master.

**Comment by thejerrytan** [comment]
as per @ser94mor comments, i am going to close this and open a new PR here:
https://github.com/ser94mor/self-driving-car-using-ros/pull/28

**Comment by ser94mor** [comment]
This pull request **introduces 2 alerts** when merging 5b5002d816303463a3dc12445c3bab8fc8fa9d11 into e9f99fe9a7e0b70bb7b75b34bf95e7d9ba46f1cf - [view on LGTM.com](https://lgtm.com/projects/g/ser94mor/self-driving-car-using-ros/rev/pr-7b3501c6f12bea196006b6dc39fc32cd9f3fbfd8)

**new alerts:**

* 2 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*
